Sources/PackageManagerDocs/Documentation.docc/PackageCollections.md

```diff
@@ -332,7 +336,7 @@ A signed package collection has an extra `signature` object:
 ```
 
 - The signature string (represented by `"<SIGNATURE>"`) is used to verify the contents of the collection file haven't been tampered with since it was signed when SwiftPM user [adds the collection](<doc:PackageCollectionAdd#Signed-package-collections>) to their configured list of collections. It includes the certificate's public key and chain.
-- `certificate` contains details extracted from the signing certificate. `subject.commonName` should be consistent with the name of the publisher so that it's recognizable by users. The root of the certificate must be [installed and trusted on users' machines](<doc:PackageCollectionAdd#trusted-root-certificates>).
+- `certificate` contains details extracted from the signing certificate. `subject.commonName` should be consistent with the name of the publisher so that it's recognizable by users. The root of the certificate must be [installed and trusted on users' machines](<doc:PackageCollectionAdd#Trusted-root-certificates>).
```

Comment:
Thanks for fixing this one too! I moved this stuff around in [my PR](https://github.com/swiftlang/swift-package-manager/pull/8745) for the registry topics, so depending on the timing of the merges there may or may not be some conflicts. Just wanted to put that out there in case I forgot :)